TypeCobol.Test/Utils/Comparator.cs

```diff
@@ -511,6 +511,19 @@ public override bool Visit(CommitStatement commitStatement)
                 //No SqlObject in CommitStatement
                 return true;
             }
+
+            public override bool Visit(TruncateStatement truncateStatement)
+            {
+                _writer.WriteLine($"line {truncateStatement.Line}: {nameof(TruncateStatement)}");
+                DumpSqlObject(nameof(truncateStatement.TableName), truncateStatement.TableName);
+                _writer.Write($"- {nameof(truncateStatement.StorageManagement)} = ");
+                _writer.WriteLine(truncateStatement.StorageManagement != null ? truncateStatement.StorageManagement.ToString() : "<NULL>");
+                _writer.Write($"- {nameof(truncateStatement.DeleteTriggersHandling)} = ");
+                _writer.WriteLine(truncateStatement.DeleteTriggersHandling != null ? truncateStatement.DeleteTriggersHandling.ToString() : "<NULL>");
+                _writer.Write($"- {nameof(truncateStatement.IsImmediate)} = ");
+                _writer.WriteLine(truncateStatement.IsImmediate != null ? truncateStatement.IsImmediate.ToString() : "<NULL>");
```

Comment:
Add a `private void DumpObject(string name, object value)` method to factorize the writing of a non-SqlObject property.